### ISSUE TITLE
UI/UX Decorative figures/images

### ISSUE THREAD

**Issue by ntderosu-code** [ntderosu-code/PDF-Assist11y#139]
### Screen

Alt Text wizard

### What's off?

We need to be able to re-tag decorative images or figures on the alt text panel. decorative images should be artifacts, per the PDF spec
